README.md

```diff
@@ -139,6 +139,29 @@ public class MainActivity extends AppCompatActivity implements WebviewListener{
 DesignhubzWebview.initializeComponents(this);
 ```
 
+- To Send userID To SDK:
+    - Before the start eyewear tryon need to send UserID to SDK then after other methods will be called.
```

Comment:
@ninjacoderrakesh  The user should call sendUserId as soon as they receive the result from the startEyewearTryon method.